Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md

```diff
@@ -0,0 +1,109 @@
+# Master Control Signoff
+## Functionality
+![Alt text](./Images/Master_Control_Pinout)
+
+The Master Control subsytem will facilitate navigation and control of the other
+	subsystems to direct the robot around the board. The system will receive 
+	inputs from the start button, sensing microcontroller, and other peripherals
+	designed by future teams, and it will output instructions to the motor Control
+	subsystem and other specified systems (designed by future teams). The subsystem will use a 
+	Jetson Nano microcontroller to do all calculations and to send instructions. 
+	The system will ultimately perform pathfinding, navigation, and user-defined peripheral tasks,
+	but for the scope of the current system, pathfinding and navigation will be addressed 
+	and user-defined peripherals will be considered and allotted for. 
```

Comment:
Is the last sentence finished? for future?

Reply:
Fixed the last sentence to make it more concise: says "allotted for future use" now